File: analysis/analysers/parsers/DependencyParser/src/test/resources/analysis/contract/examples/csharp/src/de/sots/cellarsandcentaurs/adapter/persistence/CreatureEntity.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace de.sots.cellarsandcentaurs.adapter.persistence
{
    using A = de.sots.cellarsandcentaurs.application;
    [Table("CreatureEntity")]
    public class CreatureEntity(Guid id)
    {
        [Key]
        public Guid Id { get; set; }=id;

        public CreatureEntity()
        {
        }
    }
}

File: analysis/analysers/parsers/DependencyParser/src/test/resources/analysis/contract/examples/java/src/main/java/de/sots/cellarsandcentaurs/adapter/persistence/CreatureEntity.java
package de.sots.cellarsandcentaurs.adapter.persistence;

import java.util.UUID;

public class CreatureEntity {
    private UUID id;

    public CreatureEntity(UUID id) {
        this.id = id;
    }

    public CreatureEntity() {
    }

    public UUID getId() {
        return id;
    }
}
